### ISSUE TITLE
ECMPS View Comments - Modal Dialog has incorrect Focus management

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#3593]
**Issue:**
The View Comments modal does not place the focus back on the original element when the modal is closed.

When a modal Dialog Box appears, the keyboard focus must be placed on the first actionable control. If there is no other actionable control, then focus should be placed on the close button. To ensure that the modal dialog operates correctly, code the dialog using ARIA.

**Comment by aprematta** [comment]
Focus has been fixed.
